### PR TITLE
feat(docs): modify events_disconnect_resaon of data_integrations

### PR DESCRIPTION
**Pull request by luoyq0629**

### CODE REVIEW

File: en_US/rule_engine/rule_engine_events.md [hunk 1]
```diff
@@ -67,25 +67,43 @@ You can test the SQL, view the return field after firing this rule.
...
+| event           | Event type, fixed at "client.disconnected" |
+| node            | Node name of the trigger event             |
+| peername        | IPAddress and Port of terminal             |
+| reason          | Reason for disconnection of terminal       |
+| sockname        | IPAddress and Port listened by emqx        |
+| timestamp       | Event trigger time (ms)                    |
+| username        | Current MQTT username                      |
+| peername        | IPAddress and Port of terminal             |
+
+Common client disconnection reasons are as follows.
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_events.md:83]: Here are some common reasons for client disconnections.

File: en_US/rule_engine/rule_engine_events.md [hunk 2]
```diff
@@ -67,25 +67,43 @@ You can test the SQL, view the return field after firing this rule.
...
+
+| types                    | reason                                                                                                                                                                                                                                                                |
+| ------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| normal                   | MQTT client disconnects normally                                                                                                                                                                                                                                      |
+| kicked                   | Server-side kick, via REST API                                                                                                                                                                                                                                        |
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_events.md:88]: Using the REST API to kick out clients

File: en_US/rule_engine/rule_engine_events.md [hunk 3]
```diff
@@ -67,25 +67,43 @@ You can test the SQL, view the return field after firing this rule.
...
+| keepalive_timeout        | MQTT keepalive timeout                                                                                                                                                                                                                                                |
+| not_authorized           | Authentication fails, or Pub/Sub without permission will actively disconnect the client when acl_nomatch = disconnect                                                                                                                                                 |
+| tcp_closed               | The peer has closed the network connection                                                                                                                                                                                                                            |
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_events.md:91]: The TCP connection is actively closed by the client.

File: en_US/rule_engine/rule_engine_events.md [hunk 4]
```diff
@@ -67,25 +67,43 @@ You can test the SQL, view the return field after firing this rule.
...
+| receive_maximum_exceeded | When the client sends a qos2 message, because it does not reply to PUBREL in time, a large number of PUBREC packets are accumulated on the Broker side. When the number of accumulated packets reaches the upper limit, EMQX will actively disconnect the connection. |
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_events.md:92]: Broker side -> MQTT Broker

Comment (zibuyu1995) [reply en_US/rule_engine/rule_engine_events.md:92]: accumulated -> pileup

File: zh_CN/rule_engine/rule_engine_events.md [hunk 5]
```diff
@@ -66,25 +66,44 @@ EMQX Cloud 数据集成 提供了以下七个可用的事件订阅主题。
...
+
+常见客户端离线原因如下。
+
+| 类型                     | 原因                                                                                                                         |
+| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------- |
+| normal                   | MQTT 客户端正常断开                                                                                                          |
+| kicked                   | 服务端踢出，通过 REST API                                                                                                    |
+| keepalive_timeout        | MQTT keepalive 超时                                                                                                          |
+| not_authorized           | 认证失败，或者 acl_nomatch = disconnect 时没有权限的 Pub/Sub 会主动断开客户端                                                |
+| tcp_closed               | 对端关闭了网络连接                                                                                                           |
```
Comment (zibuyu1995) [inline zh_CN/rule_engine/rule_engine_events.md:90]: 客户端主动关闭 TCP 连接

**Review by zibuyu1995 — APPROVED** [review]

**Review by chenx-work — APPROVED** [review]

File: zh_CN/rule_engine/rule_engine_events.md [hunk 6]
```diff
@@ -66,25 +66,43 @@ EMQX Cloud 数据集成 提供了以下七个可用的事件订阅主题。
...
+| disconnected_at | 终端连接断开时间 (s)                   |
+| event           | 事件类型，固定为 "client.disconnected" |
+| node            | 事件触发所在节点                       |
+| peername        | MQTT Session 过期时间                  |
+| reason          | 客户端连接断开原因                     |
+| sockname        | emqx 监听的 IPAddress 和 Port          |
+| timestamp       | 事件触发时间 (ms)                      |
+| username        | 用户名                                 |
+| peername        | 终端的 IPAddress 和 Port               |
+
```
Comment (azuregaya) [inline zh_CN/rule_engine/rule_engine_events.md:81]: 把 reason 放到最后，好理解一些

File: zh_CN/rule_engine/rule_engine_events.md [hunk 7]
```diff
@@ -66,25 +66,43 @@ EMQX Cloud 数据集成 提供了以下七个可用的事件订阅主题。
...
+常见客户端离线原因如下。
+
+| 类型                     | 原因                                                                                                                         |
```
Comment (azuregaya) [inline zh_CN/rule_engine/rule_engine_events.md:84]: Reason 类型

**Review by lucas-emqx — APPROVED** [review]

**Pull request merged**